snuba/datasets/entity.py

```diff
@@ -92,6 +93,13 @@ def get_all_storages(self) -> Sequence[Storage]:
         """
         return self.__storages
 
+    def get_valid_functions(self) -> Set[str]:
```

Comment:
seemed to make the most sense to me to have checking if the function is a valid function be separate from the actual validator of the function itself.